GoogleChromeLabs/ps-analysis-tool
# Feature: Add speed slider logic to the extension

## Description
This PR aims to add speed slider logic which enables viewers to speed-up and slow down the animation in explorable explanations for Protected Audience

## Relevant Technical Choices



## Testing Instructions
- Clone the branch
- In the terminal run `npm i && npm run build:ext`.
- Open example.com in the browser.
- Now in the browser, open DevTools and then open PSAT.
- In PSAT, navigate to protected audience section and open explorable explanations.
- You should be able to control the speed of the animation.

## Additional Information:



## Screenshot/Screencast



---

## Checklist



~~- [ ] This code is covered by unit tests to verify that it works as intended.~~ NA
~~- [ ] The QA of this PR is done by a member of the QA team (to be checked by QA).~~ NA



Partially addresses #857
